docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -1757,16 +1757,21 @@ Create an image either by pulling it from the registry or by importing it
 
 **Example request**:
 
-    POST /images/create?fromImage=ubuntu HTTP/1.1
+    POST /images/create?fromImage=ubuntu&tag=latest HTTP/1.1
 
 **Example response**:
 
     HTTP/1.1 200 OK
     Content-Type: application/json
 
-    {"status": "Pulling..."}
-    {"status": "Pulling", "progress": "1 B/ 100 B", "progressDetail": {"current": 1, "total": 100}}
-    {"error": "Invalid..."}
+    {"status":"Pulling from library/ubuntu","id":"latest"}
+    {"status":"Pulling fs layer","progressDetail":{},"id":"952132ac251a"}
+    {"status":"Pulling fs layer","progressDetail":{},"id":"82659f8f1b76"}
+    {"status":"Pulling fs layer","progressDetail":{},"id":"c19118ca682d"}
+    {"status":"Pulling fs layer","progressDetail":{},"id":"8296858250fe"}
+    {"status":"Pulling fs layer","progressDetail":{},"id":"24e0251a0e2c"}
+    {"status":"Waiting","progressDetail":{},"id":"8296858250fe"}
+    {"status":"Waiting","progressDetail":{},"id":"24e0251a0e2c"}
```

Comment:
Should we include "progress" and "progressDetail" as well in the example? FWIW, I used `?fromImage=busybox&tag=latest` (to have a reasonably small image), which gives:

```
{"status":"Pulling from library/busybox","id":"latest"}
{"status":"Pulling fs layer","progressDetail":{},"id":"8ddc19f16526"}
{"status":"Downloading","progressDetail":{"current":15881,"total":667590},"progress":"[=\u003e                                                 ] 15.88 kB/667.6 kB","id":"8ddc19f16526"}
{"status":"Downloading","progressDetail":{"current":556269,"total":667590},"progress":"[=========================================\u003e         ] 556.3 kB/667.6 kB","id":"8ddc19f16526"}
{"status":"Download complete","progressDetail":{},"id":"8ddc19f16526"}
{"status":"Extracting","progressDetail":{"current":32768,"total":667590},"progress":"[==\u003e                                                ] 32.77 kB/667.6 kB","id":"8ddc19f16526"}
{"status":"Extracting","progressDetail":{"current":491520,"total":667590},"progress":"[====================================\u003e              ] 491.5 kB/667.6 kB","id":"8ddc19f16526"}
{"status":"Extracting","progressDetail":{"current":667590,"total":667590},"progress":"[==================================================\u003e] 667.6 kB/667.6 kB","id":"8ddc19f16526"}
{"status":"Extracting","progressDetail":{"current":667590,"total":667590},"progress":"[==================================================\u003e] 667.6 kB/667.6 kB","id":"8ddc19f16526"}
{"status":"Pull complete","progressDetail":{},"id":"8ddc19f16526"}
{"status":"Digest: sha256:a59906e33509d14c036c8678d687bd4eec81ed7c4b8ce907b888c607f6a1e0e6"}
{"status":"Status: Downloaded newer image for busybox:latest"}
```